snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -443,8 +443,13 @@ def prime(self, force=False) -> None:
 
         elf_files = elf.get_elf_files(self.primedir, snap_files)
         all_dependencies = set()
+        # TODO: base snap support
+        core_path = common.get_core_path()
```

Comment:
Do you have a plan for this ie. how base snaps will be handled? Will the core snap be replaced? self.primedir is not what I'd expect to pass as a "base" or at least ambiguous. So maybe something like base_folder_path=self.primedir, base_snap_path=core_path - ideally another term than base but this would be more explicit.

Reply:
I am leaving the complex core_base_path as it is technically correct. multi base I will discuss with Gustavo live in January.

I have however renamed `base_path` to `root_path`